dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumInterceptor.kt

```diff
@@ -19,14 +23,14 @@ import okhttp3.Response
  * (url, method, status code, optional error).
  *
  * If you use multiple Interceptors, make sure that this one is called first.
- * If you also use the [TracingInterceptor], make it is called after this one.
+ * If you also use the [TracingInterceptor], make sure it is called before this one.
```

Comment:
Oh....this is something I am pretty sure they will miss. Maybe we can do a combined Interceptor later which uses the TracingInterceptor and RumInterceptor.

Reply:
My thoughts exactly, I added a task for that